qulice-findbugs/src/test/java/com/qulice/findbugs/FindBugsValidatorTest.java

```diff
@@ -129,7 +135,7 @@ public void excludesIncorrectClassFormCheck() throws Exception {
             .mock();
         final Environment env = new Environment.Mock()
             .withFile("target/classes/Foo.class", bytecode)
-            .withExcludes("Foo")
+            .withExcludes("findbugs", "Foo")
```

Comment:
@ilyakharlamov Let's put `findbugs` in a constant so we can try to avoid using `@SuppressWarnings("PMD.AvoidDuplicateLiterals")` at the start of the file